packages/opentelemetry-metrics/src/Meter.ts

```diff
@@ -93,4 +93,22 @@ export class Meter implements types.Meter {
     };
     return new GaugeMetric(name, opt);
   }
+
+  /**
+   * Provide a pre-computed re-useable LabelSet by
+   * converting the unordered LabelSet into a canonicalized
+   * set of lables, useful for pre-aggregation.
+   * @param labels user provided unordered LabelSet.
+   */
+  labels(labels: types.LabelSet): types.LabelSet {
```

Comment:
Should the method accept other forms of input as well? e.g. [str],[str] or Dict<str, str>

Reply:
I would prefer to avoid [str], [str] to avoid having to validate dupes and length mismatches and instead encourage best practices/simplicity, e.g. just pass a dictionary. I believe the type of [Labelset](https://github.com/open-telemetry/opentelemetry-js/pull/463/files#diff-2d417ce39c1d70832bfd48e0fc5a93f0R90) is already a dictionary